CI-CD/Jenkinsfile.Mable

```diff
@@ -0,0 +1,67 @@
+pipeline {
+	agent any
+	
+	tools { nodejs "node" }
+	
+	environment {
+		registryCredential = 'ecr:us-east-1:awscreds'
+		appRegistry = "947036392937.dkr.ecr.us-east-1.amazonaws.com/comosus-client-img"
+		comosusClientRegistry = "https://947036392937.dkr.ecr.us-east-1.amazonaws.com"
+		cluster = "ComosusClient"
+        service = "ComosusClientAppSVC"
+	}
+
+	stages {	
+		stage('Clear space'){
+		  steps {
+			    echo 'Cleanup docker space'
+                sh 'docker system prune -f'
+		  }
+		}
+		
+		stage('Fetch code'){
+		  steps {
+			git branch: 'develop', url: 'https://github.com/mable-pmyip/comosus-client.git'
+		  }
+		}
+		
+		stage('Install and Build'){
+		  steps {
+		    sh 'yarn install && yarn codegen && yarn build'
+		  }
+		}
+
+		stage('Test'){
+		  steps {
+			sh 'echo unit testing'
+		  }
+		}
+		
+		stage('Build App Image') {
+		   steps {
+            script {
+				dockerImage = docker.build( appRegistry + ":$BUILD_NUMBER", "-f Dockerfile.Mable .")
+			}
+		 }
+		}
+
+		stage('Upload App Image') {
+		  steps{
+			script {
+			  docker.withRegistry(comosusClientRegistry, registryCredential ) {
+				dockerImage.push("$BUILD_NUMBER")
+				dockerImage.push('latest')
+			  }
+			}
+		  }
+		}
+		
+		stage('Deploy to ecs') {
+          steps {
+            withAWS(credentials: 'awscreds', region: 'us-east-1') {
+                sh 'aws ecs update-service --cluster ${cluster} --service ${service} --force-new-deployment'
```

Comment:
I am not sure this will update the service to the latest image.

Reply:
Good question. From my understanding, this command will create a new container and that container will fetch the latest image. But I am not sure how it define the latest image, by tag? by uploaded time? I will have a test and do more research on that.

Reply:
Had a several test, seems like it will always deploy the image with the latest tag. 
Also found this link https://stackoverflow.com/a/62329417/18350806

Reply:
It would be better if you add some comments for this part. :)